xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+     </para>
+     <itemizedlist>
+      <listitem>
+       <formalpara>
+        <title>Public Registry:</title>
+        <para>
+        where everyone (usually registered) can download and
+        use images. A typical public registry is
+        <link xlink:href="https://hub.docker.com/">Docker Hub</link>.
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>Private Registry:</title>
+        <para>
+        these are accessible for particular users or from a
+        particular private network.
+        </para>
+       </formalpara>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Repository</term>
+    <listitem>
+     <para>
+      A <emphasis>repository</emphasis> is storage in a
+      <emphasis>registry</emphasis> that stores a different version of a
+      particular image. You can pull or push images from or to a repository.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.docker.cgroup">
+    <term>Control group</term>
+    <listitem>
+     <para>
+      <emphasis>Control groups</emphasis>, also called
+      <literal>cgroups</literal>, is a Linux kernel feature that allows
+      aggregating or partitioning tasks (processes) and all their children into
+      hierarchically organized groups to isolate resources.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.docker.namespace">
+    <term>Namespace</term>
+    <listitem>
+     <para>
+      &deng; uses <emphasis>namespaces</emphasis> for its containers that
+      isolates resources reserved for particular containers.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Orchestration</term>
+    <listitem>
+     <para>
+      In a production environment you typically need a cluster with many
+      containers on each cluster node. The containers must cooperate and you
+      need a framework that enables you to manage the containers automatically.
+      The act of automatic container management is called <emphasis>container
+      orchestration</emphasis> and is typically handled by Kubernetes.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+
+  <remark>toms 2019-02-04: I would suggest to define some more terms which
+  I think are important: layer &amp; tag.</remark>
+ </sect1>
+ <sect1 xml:id="sec.docker.concept.advantages_deng">
+  <title>Advantages of &deng;</title>
+  <para>
+   &deng; provides you with the following advantages:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Isolation of applications and operating systems through containers.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Near native performance, as &deng; manages allocation of resources in real
+     time.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Controls network interfaces and resources available inside containers
```

Comment:
This is ambiguous, too -> "Uses cgroups to control network interfaces and resources available inside containers."